### DISCUSSION TITLE
Wieder und wieder falsche Werte

### DISCUSSION THREAD

**Discussion by leonundjulie** [jomjol/AI-on-the-edge-device#2687] [General]
### The Problem

Hallo. Ich nutze die aktuelle Version und es kommt immer wake der vor, dass ich falsche Werte erhalte. Jetzt habe ich mal ein bisschen geklickert und stoße auf eine Diskrepanz. Die angehängten Fotos machen sicherlich deutlich was ich meine.
Im ROI ist alles sauber eingestellt. Aber unter RECOGNITION und OVERVIEW sieht man, dass die analogen Zähler nicht “sauber” markiert sind. Sind meine ROI falsch, obwohl sie richtig [aussehen?](url) Oder sind 
/RECOGNITION und OVERVIEW falsch @

### Version

release:%20v15.3.0%20(Commit%3A%203fbff0a)

### Logfile

```shell
None
```

### Screenshots

![image](https://github.com/jomjol/AI-on-the-edge-device/assets/120014597/88144048-9341-409b-a265-b1d236612e2e)
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/120014597/3a1fe9d2-955f-472c-aa7a-72afd05dd868)
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/120014597/6d4b970f-e0f4-4106-8081-f1f148745aad)
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/120014597/92c6617f-eb98-4714-913b-4e66348bddb5)

**Comment by caco3** [comment]
Du solltest das Reference Image neu erstellen, und dann auch die Alignment Marks und ROIs neu positionieren. Ev. it beim Speichern etwas schief gegangen.

Und halte dich bei en ROIs bitte an die Doku: https://jomjol.github.io/AI-on-the-edge-device-docs/ROI-Configuration/#how-to-setup-the-digit-rois-perfectly

Die Digit ROIs sollten alle in einer Linie sein!

**Comment by friedpa** [comment]
Man braucht einen guten Kontrast bei den Alignment Marks. Zwei runde schwarze Sticker z.B. oben links und unten rechts helfen da. Meiner Meinung nach sind deine AM zu knapp bei einander und der Kontrast und die Schärfe reichen nicht aus.

**Reply by leonundjulie** [reply]
Wo sollen die genau hin?

**Comment by leonundjulie** [comment]
Hallo. Ich habe beinahe alles versucht und auch mehrmals ein neues Referenzbild erstellt, neue Alligments …. Ich muss wohl ganz von vorne Anfangen. Wie funktioniert ein Factory Reset? Gruß, Andreas

**Reply by caco3** [reply]
ersetze den inhalt deines config files mit den defaults von hier: https://github.com/jomjol/AI-on-the-edge-device/blob/rolling/sd-card/config/config.ini

**Reply by leonundjulie** [reply]
Danke, das hat geklappt

**Comment by signrossi** [comment]
diesen Effekt hatte ich auch, bin fast verrückt geworden, dem Sourcecode nach wird das Bild nach den Markern gedreht.
Genau diesen Effekt sehe ich bei dem unteren Bild- oben die Kante läuft schräg und das Referenzbild ist schön waagerecht.
In der Config habe ich AlignmentAlgo = off gesetzt, damit habe ich es in den Griff bekommen.

**Comment by leonundjulie** [comment]
Hallo zusammen.

Ich habe alle guten Ratschläge umgesetzt:

- Mehr Kontrast erzeugt (insbesondere die Helligkeit der LED runtergenommen)
- Einen generellen Reset gemacht -> alles neu eingerichtet
- „AlignmentAlgo = off“ gesetzt

Die Empfehlung ABKLEBEN habe ich momentan noch weggelassen.

Dementsprechend gebe ich dem System jetzt mal beobachten. So sieht es in dieser Sekunde aus:

![IMG_0821](https://github.com/jomjol/AI-on-the-edge-device/assets/120014597/76ac5bdf-2d7d-4fa6-889c-2bdf5d5078dd)

**Reply by leonundjulie** [reply]
Frage: kann man die Bemerkung „Rate to high“ und die Konsequenzen deaktivieren. Soweit ich es bis dato beobachtet habe, bekomme ich erst wieder einen Wert gesetzt, wenn ich den PREVIOUS VALUE händisch neu gesetzt habe.

**Reply by Slider0007** [reply]
Kann es sein, dass du für die analogen Zeiger auch Digit ROIs konfiguriert hast? 
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/115730895/9427b461-4cfe-48c2-9b48-4c821829ee6f)
Du musst zwingend für analoge Zeiger auch Analog ROIs verwenden, ansonsten liest er die Zeiger falsch aus.
Das sollte dann so aussehen (grüner Rahmen): 
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/115730895/d49b2529-d53e-4a06-9d1c-91f305caf12e)

**Reply by leonundjulie** [reply]
Danke für den Hinweis. Hatte beim Neuanfang keinen Typ für die analogen Zeiger gewählt ;-) .. hatte mich über den blauen Rahmen gewundert, mehr aber auch nicht - jetzt ist es wieder grün und es werden auch die Nachkommastellen gelesen.

<img width="1028" alt="image" src="https://github.com/jomjol/AI-on-the-edge-device/assets/120014597/6b48a35a-4210-4272-a873-29751d438410">

**Reply by Slider0007** [reply]
So, jetzt noch über Settings - > Set Previous Value den Wert auf aktuellen Raw Wert setzen, dann sollte auch die Rate too high Meldung weg sein.

**Reply by leonundjulie** [reply]
War mit dem Foto vorhin zu schnell. Jetzt sieht es gut aus:
<img width="1022" alt="image" src="https://github.com/jomjol/AI-on-the-edge-device/assets/120014597/51b8a311-1ddc-4d46-ab47-ec3188aeb5aa">

**Comment by signrossi** [comment]
Das mit "Rate too high" ist so ein Ding, wenn eine gewisse Zeitlang keine Erkennung läuft, aber Verbrauch da ist, wird es irgendwann von alleine auf "Rate too high" hinaus laufen. Dann ist die Systematik in einer Sackgasse und muss von Hand via "Set Pre" daraus befreit werden

**Comment by leonundjulie** [comment]
Hallo. Ich habe noch eine Frage zu den Einstellungen, also zur Konfiguration. Habe ich die richtigen Zählertypen für digital und analog gewählt?
![IMG_0825](https://github.com/jomjol/AI-on-the-edge-device/assets/120014597/7aee7a4d-f7b9-4704-b121-1d4efd4e860b)
![IMG_0824](https://github.com/jomjol/AI-on-the-edge-device/assets/120014597/cec22d8c-cd48-49e1-8656-7435790592ae)

Ich frage, weil das aktuell gezeigt wird - es sind 238 m3 auf der Uhr und die Software sieht noch 237.
<img width="992" alt="image" src="https://github.com/jomjol/AI-on-the-edge-device/assets/120014597/544a53e2-3726-4d5a-8219-cd13839ac457">

**Reply by caco3** [reply]
es gibt kein richtig oder falsch bei den modellen, einfach ausprobieren, welches für dich besser passt, siehe Doku.

> Ich frage, weil das aktuell gezeigt wird - es sind 238 m3 auf der Uhr und die Software sieht noch 237.

Das Meter ist doch korrekt, es ist erst auf 237.9995!

**Comment by leonundjulie** [comment]
Hallo. Danke für die Tips. Grundsätzlich läuft es, aber alle 2 oder 3 Tage kommen keine Werte im iOBroker an und wenn mich direkt auf den Zähler einlogge sehe ich RATE TO HIGH. Ich setzte den Previous Wert neu und warte bis zum nächsten Fehler.

Habe ich die richtigen Voreinstellungen? Die diesbezüglichen Erklärungen erschließen sich mir nicht. Das sind meine Einstellungen
<img width="779" alt="image" src="https://github.com/jomjol/AI-on-the-edge-device/assets/120014597/973c3508-693e-425d-b4c8-ca8f335d2c92">

Hat jemand einen weiteren Tip?

**Reply by caco3** [reply]
ev. hast Du die max rate nicht richtig gesetzt?
Siehe https://jomjol.github.io/AI-on-the-edge-device-docs/FAQs/#rate-too-high-read

**Comment by AndreasPistoor** [comment]
Ich hatte denselben Fehler und habe die Expert Einstellung "Check Digit Increase Consistency" disabled. Danach funktionierte bei mir die Erkennung des Digit Sprunges jeweils von 9 auf 0 sauberer.
